test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -2985,10 +3040,11 @@ def test_cgroup_sismom_resize_fail(self):
             jid, execvnode1), starttime=stime)
         self.moms_list[1].log_match("Job;%s;pruned to exec_vnode=%s" % (
             jid, execvnode2), starttime=stime)
-        # Check that the sister mom failed to update the job
+        # Check that momsup saw that the sister mom failed to update the job
+        # This message is on momsup mom[1] but mentions sismom mom[0]
```

Comment:
Took me a second to realize momsup meant mother superior. We usually abbreviate it as MS.